composer.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "netgen/ibexa-scheduled-visibility",
+  "description": "Netgen's scheduled visibility mechanism for Ibexa CMS",
+  "type": "ibexa-bundle",
+  "license": "GPL-2.0-only",
+  "authors": [
+    {
+      "name": "Netgen",
+      "homepage": "https://netgen.io"
+    }
+  ],
+  "require": {
+    "php": ">=8.1",
+    "ibexa/core": "^4.6"
```

Comment:
We should probably support more 4.x versions than only 4.6. At the very least, we should check with CJW.

Reply:
Checked, latest version is OK :)